main/guides/getting-started/README.md

```diff
@@ -43,12 +48,29 @@ The Agoric SDK is supported on <a href="https://en.wikipedia.org/wiki/Linux">Lin
 - To launch a terminal on Linux, use the **Terminal** application.
 - To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
 
-## Install Node.js 16.19.1 or Higher
+## Install Go
```

Comment:
```suggestion
## Install Go version 1.20.3 or higher
```

ref:

https://github.com/Agoric/agoric-sdk/blob/b8d6697074b899630909440f0ea3caa519cdad44/repoconfig.sh#LL4C1-L4C22

Reply:
_comment_

We need to centralize documentation on required versions in a single spot so there's only one spot to update. Adding versions into titles, headers, comments, multiple docs, etc. leads to chronically out of date documentation.

Reply:
Where do you suggest? I can't think of a better place. For this audience, anyway. For the validator audience, we seem to put the required version info in each release. For the dapp / smart contract side of the SDK, we don't seem to do releases.

Perhaps putting it in the heading is unnecessary.

Reply:
Yeah, this page seems like a logical place to put all the versioning info.

I've updated the Go versioning info within the code blocks, and removed all versioning info from any headers.

Reply:
fwiw, https://github.com/Agoric/agoric-sdk/blob/master/repoconfig.sh is a candidate single-source-of-truth. But importing from there into the docs repo seems kinda tricky. It might be worth having a unit test to check that they're in sync.